.github/workflows/release.yml

```diff
@@ -20,28 +20,28 @@ jobs:
     steps:
       -
         name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       -
         name: Unshallow
         run: git fetch --prune --unshallow
       -
         name: Set up Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v4
         with:
-          go-version: "1.20"
+          go-version: '1.20'
       -
         name: Import GPG key
         id: import_gpg
-        uses: paultyng/ghaction-import-gpg@v2.1.0
+        uses: crazy-max/ghaction-import-gpg@v5
```

Comment:
Hashicorp had a fork of this, but have deprecated it and are sending people to crazy-max now. https://github.com/hashicorp/ghaction-import-gpg#warning-this-action-has-been-deprecated